tests/Form/FormTest.php

```
@@ -0,0 +1,1133 @@
+            'Should return a new instance when setting the attribute, ensuring immutability.',
+        );
+    }
+
+    public function testThrowInvalidArgumentExceptionWhenSettingAutocapitalize(): void
+    {
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage(
+            Message::VALUE_NOT_IN_LIST->getMessage(
+                'invalid-value',
+                'autocapitalize',
+                implode("', '", Enum::normalizeArray(Autocapitalize::cases())),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\Enum' in method 'testThrowInvalidArgumentExceptionWhenSettingAutocapitalize'. (undefined)

(StaticAccess)